Kitodo/src/test/java/org/kitodo/production/forms/dataeditor/DataEditorFormTest.java

```diff
@@ -73,16 +75,15 @@ public void shouldCheckIfMediaIsSelected() {
      */
     @Test
     public void shouldGetStructuralElementTitle() {
-        // configure data editor form
-        DataEditorForm dataEditorForm = new DataEditorForm();
-        dataEditorForm.getStructurePanel().setTitleMetadata(TITLE_METADATA_KEY);
+        // define which metadata keys to use as title
+        Collection<String> metadataKeys = Arrays.asList(TITLE_METADATA_KEY);
```

Comment:
```suggestion
        Collection<String> metadataKeys = List.of(TITLE_METADATA_KEY);
```
This can be simplified using `List.of`for lists containing only one element, which - according to my IDE - saves some memory.

Reply:
Done